kubernetes/spire/templates/agent/deployment.yaml

```diff
@@ -19,11 +19,22 @@ spec:
       labels:
         app.kubernetes.io/name: {{ template "spire.name" . }}-agent
     spec:
-      hostPID: true
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: {{ template "spire.name" . }}-agent
       initContainers:
+        - name: init-setperms
+          image: "{{ .Values.agent.init.repository }}:{{ .Values.agent.init.tag }}"
```

Comment:
could also add the imagePullPolicy to match the next init container
Same comment applies in the other charts that were changed.